openllm-python/src/openllm/_llm.py

```diff
@@ -1122,11 +1117,12 @@ def download():
     init_local: If True, it will initialize the model locally. This is useful if you want to run the model locally. (Symmetrical to bentoml.Runner.init_local())
     **attrs: The rest of kwargs will then be passed to the LLM. Refer to the LLM documentation for the kwargs behaviour
   '''
+
   if llm_config is not None:
     attrs.update({
-        'model_id': llm_config['env']['model_id_value'],
+        'model_id': attrs.get("model_id") or llm_config['env']['model_id_value'],
         'quantize': llm_config['env']['quantize_value'],
-        'serialisation': first_not_none(os.environ.get('OPENLLM_SERIALIZATION'), attrs.get('serialisation'), default=llm_config['serialisation']),
+        'serialisation': first_not_none(attrs.get('serialisation'), os.environ.get('OPENLLM_SERIALIZATION'), default=llm_config['serialisation']),
         'system_message': first_not_none(os.environ.get('OPENLLM_SYSTEM_MESSAGE'), attrs.get('system_message'), None),
         'prompt_template': first_not_none(os.environ.get('OPENLLM_PROMPT_TEMPLATE'), attrs.get('prompt_template'), None),
```

Comment:
`model_id` and `serialisation` should be determined at build time

`quantize` maybe should be determined at build time also?

`system_message` and `prompt_template` can be determined at runtime.

Reply:
I think the env here is for cases when you build a bento with a default system message

Reply:
ofc system message and prompt template can always be modified at runtime

Reply:
ye I think current order is fine